File: backend/entity/seat_chart.go
package entity

import (
    "gorm.io/gorm"
)

type SeatChart struct {
    gorm.Model
    Zone            string      `json:"zone"`
    QuantitySeat    int         `json:"quantity_seat"`
    Price           float64     `json:"price"`
    Availability    string      `json:"availability"`
    ConcertID      *uint
    Concert        Concert `gorm:"foreignKey:ConcertID"`
}

File: frontend/SUT/src/interface/SeatChart.ts
import { ConcertInterface } from './Concerts';

export interface SeatChartInterface {
  ID?: number;
  Zone?: string;
  QuantitySeat?: number;
  Price?: Float32Array;
  Availability?: string;

  ConcertID?: number; 
  Concert?: ConcertInterface;  
}
